app/templates/components/deprecation-article.hbs

```diff
@@ -3,7 +3,11 @@
     <a href="#{{id-for-deprecation @model.id @model.anchor}}" title={{@model.title}} class="toc-anchor">
       §
     </a>
-    {{@model.title}}
+    {{markdown-to-html
+      @model.title
+      extensions="no-wrapper"
+      tagName=""
```

Comment:
It renders a wrapping div by default. Not sure why markdown-to-html isn't a helper instead of a classic component :smile:.